flink-libraries/flink-table/src/main/scala/org/apache/flink/table/runtime/aggregate/DataSetSlideWindowAggReduceCombineFunction.scala

```diff
@@ -52,19 +55,35 @@ class DataSetSlideWindowAggReduceCombineFunction(
 
   private val intermediateRow: Row = new Row(keysAndAggregatesArity + 1)
 
+  protected var preAggfunction: GeneratedAggregations = _
+
+  override def open(config: Configuration): Unit = {
```

Comment:
Just want to confirm, so we can not reuse DataSetSlideWindowAggReduceGroupFunction open method? The preAggfunction of DataSetSlideWindowAggReduceCombineFunction and DataSetSlideWindowAggReduceGroupFunction are different?